metabolomics.yaml

```diff
@@ -168,6 +205,18 @@ tools:
     owner: fgiacomoni
     tool_panel_section_label: Metabolomics
 
+## MS/MS
+  - name: mspurity_spectralmatching
+    owner: computational-metabolomics
+    tool_panel_section_label: Metabolomics
+    
+  - name: metfrag
+    owner: tomnl
```

Comment:
@RJMW is it possible to also move this to computational-metabolomics?

We could than have trusted repos that auto-update ...

Reply:
@bgruening Yes, fine with me. What is the procedure to move metfrag and sirius-csifingerid to computational-metabolomics?

Reply:
Just change the .shed.yml file and let travis repush?

Reply:
So will this do it?

https://github.com/computational-metabolomics/metfrag-galaxy/commit/152b1ab39045fcd3e130cf90ce0021c522d0c874

https://github.com/computational-metabolomics/sirius-csifingerid-galaxy/commit/4e20bd2c7b22bc7e9cbd02a97cdfbdb6543c60eb

Or will we need to manually trigger it?

Reply:
In theory, this should have worked, but the tests have failed :(

https://travis-ci.org/computational-metabolomics/metfrag-galaxy/builds/640530620


You can always check the https://toolshed.g2.bx.psu.edu/ if it worked: https://toolshed.g2.bx.psu.edu/

Reply:
Hi @bgruening,

I still seem seem to be getting errors for the sirius_csifingerid wrapper (at the deploy stage in travis, see [travis pass](https://travis-ci.org/computational-metabolomics/sirius-csifingerid-galaxy)

I updated [the owner](https://github.com/computational-metabolomics/sirius-csifingerid-galaxy/commit/4e20bd2c7b22bc7e9cbd02a97cdfbdb6543c60eb) in the .yaml and have added both the group and user computational-metabolomics as admins for the tool. 

I have also updated the travis environmental variables to reference computational.metabolomics@gmail.com as the SHED_EMAIL and provided the password as the hidden variable SHED_PASSWORD.

Is there anything else I need to do? Is it possible to make the changes to the owner in the toolshed itself?

Reply:
changing the owner will create a new repo, with this owner, you can not use the other repository, it will be deprecated.

So you don't need to change anything in travis. Only the .shed.yml file and let travis repush. So computational-metabolomics needs to have access to the TTS and TS.

Reply:
So I could still use my thomas.nigel.lawson@gmail.com for the SHED_EMAIL in travis? (and corresponding password)

Reply:
I don't know, you need to use what ever user and password is behind `computational-metabolomics`

Reply:
Both metfrag and sirius-csifingerid wrappers are now in the toolshed with "computational-metabolomics" as the owner.

I have deprecated the metfrag and sirius-csifingerid wrappers owned by me "tomnl" as well 

Note: The test-toolshed has not been updated - as we are waiting for the password reset

Reply:
TTS is not important here ... Thanks a lot!